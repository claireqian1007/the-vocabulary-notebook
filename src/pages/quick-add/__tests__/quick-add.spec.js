import React from 'react'
import { shallow } from 'enzyme'

import QuickAddPage from '../'

describe('Given the QuickAddPage', () => {
  it('should match the snapshot by default props', () => {
    const wrapper = shallow(<QuickAddPage />)
    expect(wrapper).toMatchSnapshot()
  })
})
