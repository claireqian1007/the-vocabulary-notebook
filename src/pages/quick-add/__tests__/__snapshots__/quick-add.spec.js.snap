// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the QuickAddPage should match the snapshot by default props 1`] = `
<Box>
  Here will be the QuickAddPage
</Box>
`;
